pkg/common/load/load.go

```diff
@@ -218,7 +218,7 @@ func processValueFromString(f reflect.StructField, field reflect.Value, value st
 	return nil
 }
 
-func randomStr(length int) (str string) {
+func RandomStr(length int) (str string) {
```

Comment:
Since we've got three instances of this function now, let's make a new folder (spitballing: `pkg/common/util`?) that has this so we're not duplicating code here.